frontend/src/metabase/public/components/widgets/SharingPane.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
+
+export const IconContainer = styled.div`
+  width: 100px;
```

Comment:
Also fix the width to be 100px. I suspect because in the past this was rendered with
```css
box-sizing: content-box;
```

but we now default that to `border-box`